packages/amplify-category-api/package.json

```diff
@@ -55,6 +54,7 @@
     "graphql-key-transformer": "3.2.74",
     "graphql-predictions-transformer": "3.2.74",
     "graphql-relational-schema-transformer": "2.21.29",
+    "@aws-cdk/aws-apigatewayv2-alpha": "~2.114.1-alpha.0",
```

Comment:
This is the latest alpha package available. I tried updating to `aws-cdk-lib/aws-apigatewayv2`, E2E tests are not happy.